pkgs/by-name/hx/hx-lsp/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+rustPlatform.buildRustPackage {
+  pname = "hx-lsp";
+  version = "0.2.11";
+
+  src = fetchFromGitHub {
+    owner = "erasin";
+    repo = "hx-lsp";
+    tag = "0.2.11";
+    hash = "sha256-wTilbEK3BZehklAd+3SS2tW/vc8WEeMPUsYdDVRC/Ho=";
+  };
+
+  cargoHash = "sha256-dcGInrfWftClvzrxYZvrazm+IWWRfOZmxDJPKwu7GwM=";
+
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  meta = {
+    description = "lsp for helix , support snippets, actions";
+    homepage = "https://github.com/erasin/hx-lsp";
+    changelog = "https://github.com/erasin/hx-lsp/releases/tag/0.2.11";
```

Should use `finalAttrs.version` to set the tag and changelog URL. 
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
  pname = "hx-lsp";
  version = "0.2.11";

  src = fetchFromGitHub {
    owner = "erasin";
    repo = "hx-lsp";
    tag = finalAttrs.version;
    hash = "sha256-wTilbEK3BZehklAd+3SS2tW/vc8WEeMPUsYdDVRC/Ho=";
  };

  cargoHash = "sha256-dcGInrfWftClvzrxYZvrazm+IWWRfOZmxDJPKwu7GwM=";

  nativeInstallCheckInputs = [ versionCheckHook ];
  meta = {
    description = "lsp for helix , support snippets, actions";
    homepage = "https://github.com/erasin/hx-lsp";
    changelog = "https://github.com/erasin/hx-lsp/releases/tag/${finalAttrs.version}";
```

---

No